cvmfs/monitor.cc

```diff
@@ -286,13 +286,12 @@ string Watchdog::ReportStacktrace() {
 }
 
 
-void Watchdog::ReportSignalAndTerminate(
+void Watchdog::ReportSignalAndContinue(
   int sig, siginfo_t *siginfo, void * /* context */)
 {
   LogCvmfs(kLogMonitor, kLogSyslogErr,
            "watchdog: received unexpected signal %d from PID %d / UID %d",
            sig, siginfo->si_pid, siginfo->si_uid);
-  _exit(1);
```

Comment:
Should we really continue here for all signals or only for 15?

Reply:
I think it should be for all.  The name of the variable listing all the signals is `g_suppressed_signals` so I think the intention from the beginning was to suppress them all, not to exit.  We don't want the watchdog to exit until the primary process has exited.  The watchdog can still be killed with `-9`/`SIGKILL`.